src/sentry/api/endpoints/organization_discover.py

```diff
@@ -57,6 +60,23 @@ def __init__(self, *args, **kwargs):
         self.member = OrganizationMember.objects.get(
             user=self.context['user'], organization=self.context['organization'])
 
+        fields = kwargs['data'].get('fields') or []
+
+        match = next(
+            (
+                self.get_array_field(field).group(1)
```

Comment:
Can this regex ever fail to find a match because you would get an attribute error.

Reply:
no, the check on line 70 handles that